tests/manage/pv_services/test_pvc_rwx_writeable_after_pod_deletions.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import pytest
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+def verify_pod_node(pod_obj, node_name):
```

Comment:
s/verify_pod_name/verify_node_name and as commented in earlier, please move to pod class.

```python
pod.verify_node_name(node_name)
```

Reply:
@vasukulkarni the function is re-named as "verify_node_name".

Also, moved it to pod.py but not within the class. Mostly, all helper type functions were present outside the class, hence added this outside as well. 

Please let me know if its not ok.
```
 pod.verify_node_name(pod_obj, worker_nodes_list[node_count])
```